pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -335,13 +335,18 @@ class Build {
                         }
 
                         def jobParams = getAQATestJobParams(testType)
+
+                        if ("${testType}" == "extended.openjdk") {
+                            jobParams.put("PARALLEL_DEFAULT", "Dynamic")
+                            jobParams.put("NUM_MACHINES", "2")
+                        }
```

Comment:
I know this is only a draft but just wanted to ensure that the final version of this is done via config file(s).
eg [defauls.json](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/defaults.json) and maybe also overwirtable on a per spec basis in [jdk11u_pipeline_config.groovy](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/jobs/configurations/jdk11u_pipeline_config.groovy)

Reply:
I believe there will be discussion about the levels of parallel for different test category. If it will eventually same for all different platform, variant, version, or it varies per platform, variant, version.  The final design will based on that this hack is just to test if there is anything unexpected for dynamic.

Reply:
Right, thanks, Sophia. As long as it is configurable so that different farms can set different values.

Reply:
Correct. Thanks for suggestion, Adam.